### PR TITLE
Ability to list and cancel running (and queued) actions of bubuku

### LINKED ISSUE #61: Allow to list current actions on all the brokers
Right now it's impossible to list currently running actions on bubuku instances. It should be possible using cli tool and ui.

### PR DESCRIPTION
**Pull request by antban**
Closes #61 #62

### CODE REVIEW

File: bubuku/cli.py [hunk 1]
```diff
@@ -100,6 +102,91 @@ def swap_partitions(threshold: int):
...
+                    line_copy = dict(line)
+                    line_copy.update(change)
+                    table.append(line_copy)
+    if not table:
+        print('No brokers found')
+    else:
+        _print_table(table)
+
+
+@actions.command('delete', help='List all the actions on broker(s)')
```
Comment (adyach) [inline bubuku/cli.py]: sysctl

File: bubuku/cli.py [hunk 2]
```diff
@@ -100,6 +102,91 @@ def swap_partitions(threshold: int):
...
+@click.option('--action', type=click.STRING,
+              help='Action to delete')
+@click.option('--broker', type=click.STRING,
+              help='Broker id to list actions on. By default all brokers are enumerated')
```
Comment (adyach) [inline bubuku/cli.py]: help is incorrect

File: bubuku/cli.py [hunk 3]
```diff
@@ -100,6 +102,91 @@ def swap_partitions(threshold: int):
         RemoteCommandExecutorCheck.register_fatboy_slim(zookeeper, threshold_kb=threshold)
 
 
+@cli.group(name='actions', help='Work with running actions')
```
Comment (v-stepanov) [inline bubuku/cli.py]: What's this?

File: bubuku/communicate.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+_LOG = logging.getLogger('bubuku.communicate')
+
+
+def sleep_and_operate(controller, timeout: float):
+    cur_time = time.time()
+    finish = cur_time + (0.1 if timeout <= 0 else timeout)
+    while cur_time < finish:
+        try:
+            commnad = __COMMAND_QUEUE.get(block=True, timeout=finish - cur_time)
```
Comment (v-stepanov) [inline bubuku/communicate.py]: comm**na**d

File: bubuku/cli.py (hunk 3 above)
Comment (antban) [inline bubuku/cli.py]: @v-stepanov I don't fully get the question. It's group description if you run `bubuku-cli actions`

File: bubuku/cli.py (hunk 2 above)
Comment (antban) [inline bubuku/cli.py]: fixed https://github.com/zalando-incubator/bubuku/pull/64/commits/ff19b7550d2607d81cef29eccc92f04af6b4cf6d

File: bubuku/cli.py (hunk 1 above)
Comment (antban) [inline bubuku/cli.py]: @adyach Fixed https://github.com/zalando-incubator/bubuku/pull/64/commits/ff19b7550d2607d81cef29eccc92f04af6b4cf6d

File: bubuku/communicate.py (hunk 4 above)
Comment (antban) [inline bubuku/communicate.py]: @v-stepanov fixed https://github.com/zalando-incubator/bubuku/pull/64/commits/ff19b7550d2607d81cef29eccc92f04af6b4cf6d

File: bubuku/cli.py (hunk 3 above)
Comment (v-stepanov) [inline bubuku/cli.py]: What does group description do?

**Comment by v-stepanov** [comment]
:shipit:

**Pull request merged**